scripts/pylint.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+PREFIX="$(cd "$(dirname "$0")"/.. && pwd)/prefix"
+MIN_CODE_QUALITY=8.0
+
+get_code_score() {
+  if [ -n "$1" ]; then
+    # set minimum quality to user input (int/float) if provided and (5.0 <= input <= 10.0)
+    if [[ $1 =~ ^[0-9]*([\.][0-9])?$ ]]; then
+      if (( $(echo "$1 >= 5.0" | bc -l) )) && (( $(echo "$1 <= 10.0" | bc -l) )); then
+        MIN_CODE_QUALITY=$1
+      else
+        # In the future, we want code quality to be at minimum 8.0/10.0
+        echo "Code quality score must be between 5.0 and 10.0, inclusive."
+        echo "Recommended code quality score is >= 8.0."
+        exit 1
+      fi
+    else
+      echo "Code quality score must be an integer or floating point number."
+      exit 1
+    fi
+  fi
+  echo "Linting Python code with minimum quality of $MIN_CODE_QUALITY/10.0..."
+}
+
+check_pylint() {
+  if ! command -v pylint &>/dev/null; then
+    echo "pylint is not installed."
+    echo "Run 'sudo ./scripts/install-build-tools.sh' to install pylint."
+    exit 1
+  fi
+}
+
+get_code_score $1
+source .py_venv/bin/activate
```

Comment:
Why not use `activate-venv.sh`?

Reply:
I had this so I didn't have to run deactivate after the child process finished. I'll add the activate-venv.sh and a deactivate so that the venv is not active after running the script